frontend/src/components/VideoCall.jsx

```
@@ -0,0 +1,226 @@
+  const [isVideoOff, setIsVideoOff] = useState(false);
+
+  const myVideoRef = useRef();
+  const remoteVideoRef = useRef();
+  const peerRef = useRef();
+
+  useEffect(() => {
+    initializeCall();
+    return () => {
+      cleanup();
+    };
+  }, [initializeCall, cleanup]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)